pkg/cmd/issue/edit/edit.go

```diff
@@ -167,6 +171,21 @@ func editRun(opts *EditOptions) error {
 		return err
 	}
 
+	baseRepo, err := opts.BaseRepo()
```

Comment:
Pretty sure this can't work because `BaseRepo` will error if we are not in a git repo.